data/discord_messages.yaml

```diff
@@ -1,3 +1,1664 @@
+    here for your questions, ready to leap upon them the moment they appear.
+    Have a little patience. Keep in mind that repeatedly posting to get
+    attention is against the rules (flooding) and can get you muted or banned.
+spam:
+  content: >-
+    You can report spam [Directly to
+    Discord](https://support.discord.com/hc/en-us/articles/360000291932-How-to-Properly-Report-Issues-to-Trust-Safety)
+    and they can act against the account. On this server we can ban the user if
+    you provide a screenshot of the spam and the user's handle (eg @spammer#123)
+    in <#397426163649216512>.
+states:
+  content: >-
+    The [_States_
+    screen](https://my.home-assistant.io/redirect/developer_states) is your
+    truth table for what Home Assistant knows. Every entity is listed here,
+    along with the state the system knows (instead of the translated value you
+    see elsewhere), and all attributes.
+docker:
+  content: >-
+    If you're using any of the install methods that use Docker (basically
+    everything but HA Core) then Home Assistant runs in a container that only
+    shares a limited number of folders with the host (or add-ons if you use
+    those). If you're working with files outside of `/config/` or certain other
+    folders then those only exist _in that container_ and other containers won't
+    see them.
+
+
+    This is commonly experienced when using shell scripts or camera snapshot
+    services, when you create a file in `/root` or `/tmp` and then can't find it
+    in another container.
+blue:
+  content: >-
+    The [Home Assistant Blue](https://www.home-assistant.io/blue) is an
+    ODROID-N2+ in a custom case.
+security:
+  content: >-
+    If you have questions about the security bulletin the only place to find the
+    answers is in [the blog
+    post](https://www.home-assistant.io/blog/2021/01/14/security-bulletin/). If
+    the information isn't there we can't tell you, because we don't know either.
+
+
+    PS: We also don't know when it'll be updated.
+
+
+    If you have any questions, they're best asked on the linked [forum
+    thread](https://community.home-assistant.io/t/security-bulletin/268456/).
+reply:
+  content: >-
+    When using Discord's _Reply_ feature it defaults to pinging the person you
+    reply to, which can get frustrating for the target. Use _Shift + click_ on
+    the _Reply_ option, or click `@ ON` to `@ OFF` to stop this - on the right
+    side of the compose bar.
+
+
+    You have to change this every time (thank the Discord devs for that).
+ffs:
+  content: >-
+    When using Discord's _Reply_ feature it defaults to pinging the person you
+    reply to, which can get **very frustrating for the target**. Use _Shift +
+    click_ on the _Reply_ option, or click `@ ON` to `@ OFF` to stop this - on
+    the right side of the compose bar - you have to do this on every reply.
+
+
+    Or don't, and just wait for the recipient to get annoyed enough to block
+    you.
+raccoon:
```

Comment:
Can we hide commands from auto-complete?

And maybe we can also drop some. (in a future PR maybe). Do we really need bot commands to promote individual repo's?

Reply:
Yes
No

Reply:
https://github.com/home-assistant/service-hub/commit/f6057187bfc0b5759653a09cdd3e4acb0680e8fe hides commands from auto complete if they are missing a description (all imported commands are currently missing that)